config/eslint.cjs

```diff
@@ -80,7 +80,7 @@ module.exports = {
     'import/namespace': 'off',
     'import/no-absolute-path': 'error',
     'import/no-anonymous-default-export': 'error',
-    'import/no-cycle': 'error',
+    'import/no-cycle': 'off', // TODO: Evaluate if this is useful. We have a lot of these errors b/c of exports/imports from index.ts files
```

Comment:
I think we can get around this error by explicitly importing files (directly pointing to the file we want to import, not point to the dir). At least, that's what I assume is causing this error?